packages/config/config/devices/0x019b/tf016.json

```
@@ -211,6 +230,7 @@
 			"#": "12",
 			"$if": "firmwareVersion >= 1.8",
 			"label": "P setting",
+			"description": "Power mode. 0 is always off, 10 is always on.",
```

```suggestion
			"label": "Power mode",
```
and convert _0 is always off, 10 is always on_ into options.